src/components/tx/SignOrExecuteForm/index.tsx

```diff
@@ -234,6 +251,7 @@ const SignOrExecuteForm = ({
           nonceReadonly={nonceReadonly}
           onFormSubmit={onAdvancedSubmit}
           gasLimitError={gasLimitError}
+          willRelay={shouldExecute && willRelay}
```

Comment:
I believe you can now remove `shouldExecute` since it is already part of `willRelay`

Reply:
you're right. I've removed the redundancy